massa-grpc/src/stream/send_operations.rs

```diff
@@ -54,6 +56,29 @@ pub(crate) async fn send_operations(
                         )
                         .await;
                     } else {
+                        let Ok(now) = MassaTime::now() else {
+                            report_error(
+                                tx.clone(),
+                                tonic::Code::InvalidArgument,
+                                "too many operations per message".to_owned(),
+                            )
+                            .await;
+                            continue;
+                        };
+                        let Ok(last_slot) = get_latest_block_slot_at_timestamp(
+                            config.thread_count,
+                            config.t0,
+                            config.genesis_timestamp,
+                            now,
+                        ) else {
+                            report_error(
+                                tx.clone(),
+                                tonic::Code::InvalidArgument,
+                                "too many operations per message".to_owned(),
+                            )
```

Comment:
I don't understand the error message here and above

Reply:
fixed. Shitty copy:paste sorry